   
  <h1><center><span style = "color:#FF0000;">Titanic Data Set</span></center></h1>
  
  ***********

<h3>Background on the Titanic disaster</h3>

On April 14, 1912, The RSM Titanic struck an iceburg and sunk. By 2 am on the 15th, the ship had completely disappeared beneath the Atlantic Ocean.  Of the 2223 people abord the ship, more than 1500 passengers and crew members lost their lives. Contributing to this tragedy is that many lives were lost needlessly. The ship was equipped to carry **64 lifeboats, each with a capacity of 65 people.** Instead, the RSM Titanic made her maiden voyage with only 20 lifeboats. Even still, the 20 lifeboats could have saved more than half of the passengers and crew, but like the first lifeboat which carried only 28 people, they were not filled to capacity. This resulted in **only 31.6 percent of those on board surviving.** 

The wreckage now covers approximately **1,000 acres ** of the ocean bed.

<p>  
![Titanic Sketch](http://www.icyousee.org/graphics/titanicsketch.jpg)
<p>  

source: 
+ http://www.titanicfacts.net/
+ http://www.icyousee.org/titanic.html#serv


### Investigate a Dataset

The purpose of this analysis is to predict the factors (variables) that increased the survivability on the Titanic. I hypothesize that sex, socio-economic status and age will all play a factor a passenger’s survivability.  The data set is not a complete representation of the passengers on the Titanic but I will test if it is a good sample. Also, there is missing data. Due to this, the findings will be tentative, at best. A more complete data set is necessary to report more conclusively on survivability. 

**************

#### Using the data, the following questions will be analyzed: 

   + Does Socio-Economic status have an effect of survivability?  
   + Question 2: Does age have an effect of survivability?  
   * Question 3: Does sex have an effect of survivability?

In [3]:
## Import Python Packages
## Data Analysis
import numpy as np
import pandas as pd

## Visuals
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

In [4]:
##Import the Titanic Dataset
df = pd.read_csv("titanic-data.csv")

According to [Titanic Facts](www.titanicfacts.net), 31.6% of the total people on board survived. First I will compare this to the percent of people that survived in the sample to see if it is a accurate sample.

In [5]:
df.Survived.value_counts()

0    549
1    342
Name: Survived, dtype: int64

In [7]:
342/891.0

0.3838383838383838

** In this sample, 38% survived. I would conclude this is a good sampling of the people aboard. **